packages/next/server/base-server.ts

```diff
@@ -360,6 +362,12 @@ export default abstract class Server {
       },
     })
     this.responseCache = new ResponseCache(this.incrementalCache)
+    this.imageResponseCache = new ResponseCache(
+      new ImageOptimizerCache({
+        distDir: this.distDir,
+        nextConfig: this.nextConfig,
+      }) as any
```

Comment:
Can we do this without `as any`?

Reply:
Yeah updated the `response-cache` to be more generic so this doesn't need to be any